aten/src/ATen/native/TensorFactories.cpp

```diff
@@ -14,6 +14,7 @@
 #include <ATen/detail/CUDAHooksInterface.h>
 #include <c10/util/Exception.h>
 #include <ATen/NamedTensorUtils.h>
+#include <ATen/native/UnaryOps.h>
```

Comment:
Are there any uses of this header in this file?

Reply:
yeah, I moved `kaiser_window_stub` into UnaryOps.h because it's defined in one of the CUDA unary ops files, and then I had to fixup it here. This one probably shouldn't be using stub though.